user-managerd.pro

```diff
@@ -15,7 +15,7 @@ DEFINES += QT_DEPRECATED_WARNINGS
 # You can also select to disable deprecated APIs only up to a certain version of Qt.
 #DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
 
-PKGCONFIG += libuser glib-2.0 sailfishaccesscontrol
+PKGCONFIG += libuser glib-2.0 sailfishaccesscontrol libsystemd-login mce-qt5
```

Comment:
I wonder if that libsystemd-login was one of those deprecated libraries that were merged to the main library.

Reply:
Apparently yes.
https://github.com/systemd/systemd/blob/master/NEWS#L7915
So this should probably be without -login suffix. Please check that it builds still if that is changed.

Reply:
Removed. still seems to build.